### PR TITLE
Task/DES-1356 - Revise "Add Missions" modal for FR

### PR DESCRIPTION
**Pull request by Strmiska**
**Important:** We are removing a metadata field `elevation` from FR mission forms. `dateEnd` is being changed from **required** to **optional**

### CODE REVIEW

File: designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.js [hunk 1]
```diff
@@ -249,9 +248,7 @@ class ManageFieldReconMissionsCtrl {
         this.data.editMission.value.dateStart = new Date(
             this.data.editMission.value.dateStart
         );
-        this.data.editMission.value.dateEnd = new Date(
-            this.data.editMission.value.dateEnd
-        );
+        this.data.editMission.value.dateEnd = (this.data.editMission.value.dateEnd && this.data.editMission.value.dateEnd !== 'None') ? new Date(this.data.editMission.value.dateEnd) : '';
```
Comment (fnets) [inline designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.js]: Our linters would probably flag this line.  Can we break it into multiple lines, or make it simpler?

Comment (Strmiska) [reply designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.js]: Yeah, it's... long. The variables themselves are just too long! 😞 I'll throw it into an if else statement.

**Comment by Strmiska** [comment]
Needed to change `null` field to `''` for cleared `endDate` when an end date had been previously saved but needs to be removed.

File: designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.html [hunk 2]
```diff
@@ -146,7 +155,7 @@ <h4 class="modal-title" style="border:none;">
                     &nbsp;
                 </textarea>
             </div>
-            <div class="expAuthors form-group">
+            <!-- <div class="expAuthors form-group">
```
[the remark this answers is not in the document]
Comment (Strmiska) [reply designsafe/static/scripts/projects/components/manage-field-recon/missions/manage-field-recon-missions.component.html]: Ah yes, need to delete that.

**Comment by Strmiska** [comment]
Should be good now

**Pull request merged**